### PR TITLE
VFB-24 View Client Page

### PR DESCRIPTION
**Pull request by SamYap-Softwire**
/clients/[id] now displays the client information with the matching primary key
(But this needs to wait for Edit Client Page to be merged)

### Please prioritise reviewing Edit Client Form before reviewing this 

**If you are reviewing this, please only review files in the `/clients/[id]` folder**

- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've linted via `npm run lint`
    - can run `npm run lint_fix` to try and fix as any mistakes automatically as possible!
- [ ] Make sure you've tested via `npm run test`

### CODE REVIEW

File: src/app/clients/[id]/ViewClientPage.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const ViewClientPage: React.FC<Props> = ({ clientID, clientData }) => {
+    return (
+        <CenterComponent>
+            <StyledCard>
+                <TempDiv>
+                    {Object.entries(clientData).map(([objectKey, objectValue], index) => (
+                        <TempDiv key={index}>
+                            <KeyText>{formatCamelCaseKey(objectKey)}</KeyText>
+                            <NormalText>{objectValue as string}</NormalText>
```
Comment (SamYap-Softwire) [inline src/app/clients/[id]/ViewClientPage.tsx:42]: why.... `as string`???

File: src/app/clients/[id]/ViewClientPage.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { CenterComponent, StyledCard } from "@/components/Form/formStyling";
+import { ClientData } from "@/app/clients/[id]/getClientData";
+import { formatCamelCaseKey } from "@/pdf/ShoppingList/dataPreparation";
+
+interface Props {
+    clientID: string;
+    clientData: ClientData;
+}
+
+const NormalText = styled.pre`
```
Comment (ChloeYiuSoftwire) [inline src/app/clients/[id]/ViewClientPage.tsx:15]: Just curious, why pre not like div or p

**Review by ChloeYiuSoftwire — COMMENTED** [review]
I think the Paper storing the Child 1 Child 2 UI is not what it originally looks like - might have to be fixed

Also I think there is some bug:
Capture

File: src/app/clients/[id]/ViewClientPage.tsx (hunk 2 above)
Comment (SamYap-Softwire) [reply src/app/clients/[id]/ViewClientPage.tsx:15]: for the newlines to be rendered

**Comment by SamYap-Softwire** [comment]
it's now in a different PR :)

**Pull request closed without merging**